src/gtc/daceir.py

```diff
@@ -436,7 +439,7 @@ def set_interval(
         return GridSubset(intervals=intervals)
 
     @classmethod
-    def from_gt4py_extent(cls, extent: gtc.definitions.Extent):
+    def from_gt4py_extent(cls, extent: Extent):
```

Comment:
I actually prefer seeing the `gtc.definitions.Extent` here, but we'll take care of that in an upcoming PR :)